core/extension/gdextension_compat_hashes.cpp

```diff
@@ -61,6 +62,13 @@ bool GDExtensionCompatHashes::get_legacy_hashes(const StringName &p_class, const
 	bool found = false;
 	for (const Mapping &mapping : *methods) {
 		if (mapping.method == p_method) {
+			if (p_check_valid) {
+				MethodBind *mb = ClassDB::get_method_with_compatibility(p_class, p_method, mapping.current_hash);
+				if (!mb) {
+					WARN_PRINT(vformat("Compatibility hash %d mapped to non-existent hash %d. Please update gdextension_compat_hashes.cpp.", mapping.legacy_hash, mapping.current_hash));
```

Comment:
One point of potential confusion here is that we use hexadecimal representation of the hash in the other check, didn't realize previously, might be worth improving to unify

Reply:
Yeah, however, we use plain integers in the source code, so if someone is trying to find the hash in the code it's more useful in integer form. It would be good to unify this in some way, though.